docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Session Connectivity Resource Disonnection during a Session"
+linkTitle: "Session Connectivity"
+weight: 20
+type: "docs"
+description: "How to troubleshoot a resource losing connection during a robot session."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
+---
+
+A session:
+
+- allows a client to express that it is actively connected or authenticated to `viam-server` on your robot
+- supports stopping moving components when the connection is no longer active.
+
+### how does viam-server work if you lose connectivity during a session?
```

Comment:
Will be changing this and making these headings a bit more formal when this is not a draft